lib/cdo/safe_browsing.rb

```diff
@@ -52,6 +52,8 @@ def self.determine_safe_to_open(url_to_check)
 
     # Safe Browsing API returns empty JSON object is no threat matches found
     # Do not determine safe if response is nil
-    response.nil? ? false : response_value
+    # Determine safe if bad request (response code is 400) or if rate-limited (429)
+    error_response = response.nil? ? false : (response.code === '400' || response.code === '429')
```

Comment:
Is `==` more appropriate as a comparison operator here instead of `===`?  
I didn’t even realize Ruby had an `===` operator, so I had to [look up](https://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and) how it’s used.

Reply:
Good call - I just defaulted to JavaScript equality at first. Updated.